src/content/structured/get-started/test-app/index.ts

```diff
@@ -0,0 +1,1817 @@
+const testFiles = {
+  cypress: {
+    fileName: "cypress.cy.tsx",
+    disableMoreButton: false,
+    snippets: {
+      short: `it("should fill the form with no errors/validation", () => {
+            mount(<Subscription />);
+            // checkHydrated will wait until the component is hydrated and ensures it is ready to test
+            checkHydrated(IC_PAGE_HEADER);
+
+            // Check the first step is visible
+            cy.get(IC_STEPPER).should(BE_VISIBLE);
+            checkCurrentStep(0);
+
+            // Select a radio option
+            findShadowEl(IC_RADIO_OPTION, RADIO+'[value="house"]').check({ force: true });
+            cy.get(IC_RADIO_OPTION+'[label="House Blend"]').should(HAVE_ATTR, SELECTED);
+
+            // Select a select option using mouse
+            clickOnShadowEl(IC_SELECT, IC_INPUT_CONTAINER);
+            clickOnShadowEl(IC_SELECT, IC_MENU_OPTION+'[data-value="aeropress"]');
+
+            // Select an option using keyboard
```

Comment:
For consistency with the previous comment, change comment to "Select a select option using keyboard".